packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -2149,6 +2150,12 @@ class CupertinoDialogAction extends StatefulWidget {
   /// [_kCupertinoDialogActionStyle].
   final TextStyle? textStyle;
 
+  /// The cursor that will be shown when hovering over the button.
+  ///
+  /// If null, defaults to [SystemMouseCursors.click] on web and
+  /// [MouseCursor.defer] on other platforms.
```

Comment:
should also mention this is ignored if the button is disabled. ie. onPressed == null